### PR TITLE
Editing dlgMetadata

### PR DESCRIPTION
**Pull request by Lunalo**

### CODE REVIEW

File: instat/dlgMetadata.vb [hunk 1]
```diff
@@ -50,13 +50,14 @@ Public Class dlgMetadata
...
-        Me.Hide()
         If ucrDFSelectorForMetadata.cboAvailableDataFrames.SelectedItem = "[Overall]" Then
             frmMetaData.Show()
+            frmMetaData.BringToFront()
         Else
             frmVariables.Show()
+            frmVariables.BringToFront()
         End If
-
+        Me.Close()
```
Comment (dannyparsons) [inline instat/dlgMetadata.vb]: I think Hide was fine, why did you change to close?

Comment (Lunalo) [inline instat/dlgMetadata.vb]: Hi Danny,


Me.close terminates the running instance while Me.Hide just hide the dialog.

We had agreed that the dialog was supposed to be hidden without running
anything

Regards,
Mr. John Ong'ala Lunalo|| Kenyatta University Alumnus ||
Volunteer, African Maths Initiative || Software Developer, African Data
Initiative ||
Email Address :johnlunalo95@gmail.com/ johnlunalo@yahoo.com
Tel: +254701519572, +254764000861|| Skype contact: john.lunalo

Comment (dannyparsons) [inline instat/dlgMetadata.vb]: Yes I agree. So why are you using Close?
